edx_proctoring/static/proctoring/js/views/proctored_exam_view.js

```diff
@@ -85,12 +85,12 @@ var edx = edx || {};
                               action: 'stop'
                             },
                             success: function() {
-                              // Reloading page will reflect the new state of the attempt
-                              location.reload();
+                              // change the location of the page to the active exam page
+                              // which will reflect the new state of the attempt
+                              location.href = self.model.get('exam_url_path');
```

Comment:
I have checked on the firefox, chrome and safari browsers and it is working fine(reloading the page to the exam url).

for browser support I have also looked at the following link http://www.w3schools.com/jsref/prop_loc_href.asp